### ISSUE TITLE
Data discrepancies

### ISSUE THREAD

**Issue by ulfelder** [christophergandrud/imfr#6]
This issue might have nothing to do with `imfr` and everything to do with the IMF's API or databases, but I thought I would ask in case:

I am currently working with the IMF's monthly data on the consumer price index (CPI). Here is the code I am using to load this data set into R:
```
library(imfr)
DF <- imf_data(database_id = 'CPI',
               indicator = 'PCPI_IX',
               start = 1960,
               end = lubridate::year(Sys.Date()),
               freq = 'M')
```
The code works fine. However, when I compare the data for Mauritius (`DF[DF$iso2c=="MU",]`) to the data a colleague got when he did a "bulk download" of the same database from the IMF's web page, we found that they differ. Specifically, the data I got through `imf_data` show several sharp breaks in the series -- what look like re-baselinings -- whereas the data from the bulk download do not show those breaks.

Can you think of any way this might be caused by something in `imfr` and not a glitch in the IMF API or data-management practices?

In case it matters, I am using version 0.1.1 of `imfr` downloaded from CRAN in version 3.2.4 of R for 64-bit Windows systems.

I'm guessing it's an IMF problem, not an `imfr` one, but I thought you would be interested to hear about it either way.

**Comment by christophergandrud** [comment]
I can't think of anything. One way to check would be to run your code above with the argument `print_url = TRUE`. Then copy and paste the returned URL into your browser. This will take you to the original JSON, before *imfr* has done anything to it. 

If the unexpected phenomena are in that, they odds are the issue is on the IMF's side. (The bulk download and API accessible data could just be different, for example).

**Comment by ulfelder** [comment]
Thank you. I tried what you proposed for inspecting the original JSON, looking only at the series for Mauritius ([this url](http://dataservices.imf.org/REST/SDMX_JSON.svc/CompactData/CPI/.MU.PCPI_IX?startPeriod=1960&endPeriod=2016)). The data visible there show an additional column, REFERENCE_PERIOD, that isn't returned by `imf_data`. That column appears to show that the data for Mauritius are re-baselined several times, with the periods matching the breaks I saw in my line plot. So it looks like the data available via bulk download do differ from the data available through the API; the former have been put on a common baseline, while the latter have not.

So, as suspected, this isn't an `imfr` issue.

I wonder why the IMF would make different versions of their data available through those two paths, especially without flagging the fact that they are doing this?
